server/utils/sorters.ts

```diff
@@ -14,3 +14,8 @@ export const identifiedNeedSorter = (
   a.createdDate.localeCompare(b.createdDate) || // 2. sort by ascending Create Date, if tie then
   a.identifiedNeed.localeCompare(b.identifiedNeed) || // 3. sort by title alphabetically, if tie then
   a.identifiedNeedUuid.localeCompare(b.identifiedNeedUuid) // 4. sort by UUID alphabetically
+
+export const attendeeSorter = (a: components['schemas']['Attendee'], b: components['schemas']['Attendee']) =>
+  Number(!a.name) - Number(!b.name) || // 1. sort nullish name to bottom, if tie then
+  (a.name && b.name && a.name.localeCompare(b.name)) || // 2. sort by non-null name alphabetically, if tie then
+  a.attendeeUuid.localeCompare(b.attendeeUuid) // 3. sort by UUID alphabetically
```

Comment:
Final tie break by created date would be sufficient here I think

Reply:
multiple attendees on the same review would be created at the same time, i assume they might have the exact same timestamp? that's why i think tiebreak with uuid would give the best certainty